.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+---
+version: 2
+updates:
+    - package-ecosystem: pip
+      directory: /
+      schedule:
+          interval: monthly
+    # Raise pull requests for version updates
+    # to pip against the `develop` branch
+      target-branch: develop
+    # Labels on pull requests for version updates only
+      labels:
+          - pip dependencies
```

Comment:
Is this label right? It won't always be `pip` and will often be `npm`.

I think the right label is `misc`

Reply:
I will update label to npm